dbt/include/athena/macros/materializations/seeds/helpers.sql

```diff
@@ -14,18 +14,27 @@
   {%- set s3_data_naming = config.get('s3_data_naming', target.s3_data_naming) -%}
   {%- set external_location = config.get('external_location', default=none) -%}
 
+  {%- set s3_location = adapter.upload_seed_to_s3(
+    s3_data_dir,
+    s3_data_naming,
+    external_location,
+    model.schema,
+    model.name,
+    agate_table,
+  ) -%}
+
   {% set sql %}
-    create external table {{ this.render_hive() }} (
+    CREATE EXTERNAL TABLE {{ this.render_hive() }} (
```

Comment:
We use lowercase everywhere, could we stick with that?

Reply:
@nicor88 👍 updated